src/meta/src/controller/mod.rs

```diff
@@ -67,21 +67,39 @@ pub struct SqlMetaStore {
 impl SqlMetaStore {
     /// Connect to the SQL meta store based on the given configuration.
     pub async fn connect(backend: MetaStoreBackend) -> Result<Self, sea_orm::DbErr> {
+        const MAX_DURATION: Duration = Duration::new(u64::MAX / 4, 0);
```

Comment:
what's the story behind the num?

Reply:
Nothing complicated. I just encountered panics due to arithmetic overflow with `MAX` or `MAX / 2` in library's code.